docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -8,6 +8,18 @@
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td><h5>as-of-snapshot</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>Long</td>
+            <td>Read snapshot specific by snapshot id.</td>
+        </tr>
+        <tr>
+            <td><h5>as-of-timestamp-mills</h5></td>
```

Comment:
Why introduce a new configuration? Time travel reading is actually very similar to the "from-timestamp" startup mode in streaming jobs. I'm considering supporting startup mode for both batch and streaming jobs. See https://issues.apache.org/jira/browse/FLINK-30294 .